index.html

```diff
@@ -0,0 +1,162 @@
+                    <li><a href="#"><i class="fa brand fa fa-angellist" ></i></a></li>
+                </ul>
+            </div>
+        </section>
+   <!--first card-->
+   
+   <!--work-section-->
+<section class="card">
+    <img class="first-image" src="/images/1.png" alt="">
+    <h1 class="title">Tonic</h1>
+
+        <ul class="field">
+            <li class="cano">CANOPY</li>
+            <li class="tag">&#8226; Back End Dev </li>
+            <li class="tag">&#8226; 2015</li>
+        </ul>
+    <p class="instruction"> A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+    <ul class="trainning">
+        <li class="other-skills">html</li>
+        <li class="other-skills">css</li>
+        <li class="other-skills">javascript</li>
+    </ul>
+    <button class="card-btn">See Project</button>
+</section>
+<section class="card">
+    <img class="first-image" src="/images/2.png" alt="">
+    <h1 class="title">Multi-Post
+        Stories</h1>
+
+        <ul class="field">
+            <li class="cano">CANOPY</li>
+            <li class="tag">&#8226; Back End Dev </li>
+            <li class="tag">&#8226; 2015</li>
+        </ul>
+    <p class="instruction"> A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+    <ul class="trainning">
+        <li class="other-skills">html</li>
+        <li class="other-skills">css</li>
+        <li class="other-skills">javascript</li>
+    </ul>
+    <button class="card-btn">See Project</button>
+</section>
+<section class="card">
+    <img class="first-image" src="/images/3.png" alt="">
+    <h1 class="title">Tonic</h1>
+
+        <ul class="field">
+            <li class="cano">CANOPY</li>
+            <li class="tag">&#8226; Back End Dev </li>
+            <li class="tag">&#8226; 2015</li>
+        </ul>
+    <p class="instruction"> A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+    <ul class="trainning">
+        <li class="other-skills">html</li>
+        <li class="other-skills">css</li>
+        <li class="other-skills">javascript</li>
+    </ul>
+    <button class="card-btn">See Project</button>
+</section>
+<section class="card">
+    <img class="first-image" src="/images/4.png" alt="">
+    <h1 class="title">Multi-Post
+        Stories</h1>
+
+        <ul class="field">
+            <li class="cano">CANOPY</li>
+            <li class="tag">&#8226; Back End Dev </li>
+            <li class="tag">&#8226; 2015</li>
+        </ul>
+    <p class="instruction"> A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+    <ul class="trainning">
+        <li class="other-skills">html</li>
+        <li class="other-skills">css</li>
+        <li class="other-skills">javascript</li>
+    </ul>
+    <button class="card-btn">See Project</button>
+</section>
```

Comment:
- [ ] kindly consider wrapping the whole work section into a single section tag this will help you to follow HTML and CSS best practices
- [ ] after that remember that In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.kindly fix tat as well